params/params.yaml

```diff
@@ -1,7 +1,42 @@
 amcl:
   ros__parameters:
-    laser_max_range: -1.0 # Use value reported in laser scan
-    laser_min_range: -1.0 # Use value reported in laser scan
+    alpha1: 0.8  # Expected noise in odometry rotation estimate from rotation
+    alpha2: 0.4  # Expected noise in odometry rotation estimate from translation
+    alpha3: 0.2  # Expected noise in odometry translation estimate from translation
+    alpha4: 0.3  # Expected noise in odometry translation estimate from rotation
+    alpha5: 0.2
+    base_frame_id: "base_footprint"
+    beam_skip_distance: 0.5
+    beam_skip_error_threshold: 0.9
+    beam_skip_threshold: 0.3
+    do_beamskip: false # related to laser_model_type for prob. est
+    global_frame_id: "map"
+    lambda_short: 0.1 # exponential decay rate of z_short
+    laser_likelihood_max_dist: 2.0
+    laser_max_range: 10.0  # negative to match internal limits
+    laser_min_range: 0.1   # negative to match internal limits
```

Comment:
Is `negative to match internal limits` carry-over from a previous parameter setting? Either way, I think it would be worth re-wording to make this more clear.